app/containers/CvEdit/index.js

```diff
@@ -144,9 +146,10 @@ export class CvEdit extends React.Component {
     const items =
       section.items.map((item, i) => this.renderItem(item, index, i, section))
     const addItemEnabled = section.items.length < MAX_ITEMS
-    const addItemClasses = addItemEnabled
+    const addItemClasses = classNames(addItemEnabled
       ? 'btn-bright'
-      : 'btn-disabled'
+      : 'btn-disabled', styles.cvAction)
```

Comment:
Replacing ternarys with passing an object instead would make it much more readable imo.
e.g:
```
const classes = classNames({
  'btn-bright': addItemEnabled,
  'btn'disabled': !addItemEnabled,
})
```